301-multi-tier-loadbalancing/azuredeploy.json

```diff
@@ -0,0 +1,1073 @@
+      "dependsOn": [],
+      "properties": {
+        "publicIPAllocationMethod": "Dynamic",
+        "dnsSettings": {
+          "domainNameLabel": "[toLower(concat(variables('region-tm-test-vm'),copyIndex()))]"
+        },
+        "idleTimeoutInMinutes": 4
+      },
+      "copy": {
+        "name": "region-tm-vm-pips-loop",
+        "count": 2
+      },
+      "resources": []
+    },
+    {
+      "apiVersion": "2016-03-30",
+      "name": "[concat(variables('region-tm-vm-nic'),copyIndex())]",
+      "type": "Microsoft.Network/networkInterfaces",
+      "location": "[variables('region')[copyIndex()]]",
+      "tags": {
+        "displayName": "region-tm-vm-nics"
+      },
+      "comments": "Test VMs to test traffic manager scenario from each region!",
+      "dependsOn": [
+        "vnetloop",
+        "region-tm-vm-pips-loop"
+      ],
+      "properties": {
+        "ipConfigurations": [
+          {
+            "name": "ipconfig1",
+            "properties": {
+              "privateIPAddress": "10.0.4.4",
+              "privateIPAllocationMethod": "Dynamic",
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses',concat(variables('region-tm-vm-pip'),copyIndex()))]"
+              },
+              "subnet": {
+                "id": "[concat(resourceId('Microsoft.Network/virtualNetworks', concat(variables('region-vnet-name'),copyIndex())), '/subnets/default')]"
+              }
+            }
+          }
+        ],
+        "dnsSettings": {
+          "dnsServers": []
+        },
+        "enableIPForwarding": false
+      },
+      "copy": {
+        "name": "region-tm-vm-nics-loop",
+        "count": 2
+      },
+      "resources": []
+    },
+    {
+      "apiVersion": "2015-06-15",
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[concat(variables('region-tm-test-vm'),copyIndex())]",
+      "location": "[variables('region')[copyIndex()]]",
+      "tags": {
+        "displayName": "regions-test-vms"
+      },
+      "comments": "Test VMs to test traffic manager scenario from each region!",
+      "dependsOn": [
+        "region-tm-vm-pips-loop",
+        "storage-accounts-loop",
+        "region-tm-vm-nics-loop"
+      ],
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "Standard_DS1_v2"
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "MicrosoftWindowsServer",
+            "offer": "WindowsServer",
+            "sku": "2012-R2-Datacenter",
+            "version": "latest"
+          },
+          "osDisk": {
+            "name": "[concat(variables('region-tm-test-vm'),copyIndex())]",
+            "createOption": "FromImage",
+            "vhd": {
+              "uri": "[concat('https', '://', variables('region-storage-account'),copyIndex(),'.blob.core.windows.net', concat('/vhds/',variables('region-tm-test-vm'),copyIndex(),'.vhd'))]"
```

Comment:
Please see the best practice around 'uris compatable with all clouds': https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md